engine/src/main/java/org/camunda/bpm/engine/impl/telemetry/dto/Internals.java

```diff
@@ -31,6 +31,7 @@
   protected Database database;
   @SerializedName(value = SERIALIZED_APPLICATION_SERVER)
   protected ApplicationServer applicationServer;
+  protected LicenseKeyData licenseKey;
```

Comment:
Let's adjust the name of the field according to the ticket description
```suggestion
  @SerializedName(value = "license-key")
  protected LicenseKeyData licenseKey;
```
```suggestion
  protected LicenseKeyData licenseKey;
```